keps/prod-readiness/sig-node/2238.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 2238
 alpha:
   approver: "@johnbelamaric"
+beta:
+  approver: "@johnbelamaric"
```

Comment:
Looks like Elana approved this, so change beta approver to her. Thanks!

Reply:
@johnbelamaric it's my KEP, I can't review my own KEP :joy:

Reply:
Doh! I should have known better, I remember now from last cycle this one is yours :)